scripts/consent_form_json.py

```diff
@@ -1,24 +1,90 @@
+""" Download all csv files with consent form content from google drive and combine into a single json file.
+    Requires a client_secret.json file with developer application credentials from google drive.
```

Comment:
Once this feature stabilizes, can we document feature requirements in a central location, such as `README.md`?

This would include:
- Scripts run to set up part of the application
- Files required for application or scripts to work
- What the contents of the file should look like